.gitignore

```diff
@@ -48,3 +48,5 @@ MANIFEST
 
 # ignore ISMN metadata files as they are different on win/linux
 */python_metadata/*
+
+local_scripts/*
```

Comment:
Is this necessary for the package?

Reply:
No it's just for me :) Any for anyone else who wants a location in the package to collect test files that should not be tracked